app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:show, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :set_event, only: [:show, :edit, :update]
```

Rails 8からだっけな、存在しないアクションをbefore_actionに指定すると例外扱いにするオプションあったかも :memo:

Reply:
Rails 7.1 からだった :link:
https://techracho.bpsinc.jp/hachi8833/2023_10_02/135224#rt34